content/shell/renderer/shell_content_renderer_client.cc

```diff
@@ -270,6 +271,60 @@ ShellContentRendererClient::CreateURLLoaderThrottleProvider(
   return std::make_unique<ShellContentRendererUrlLoaderThrottleProvider>();
 }
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+media::SupportedCodecs GetStarboardEmeSupportedCodecs() {
+  media::SupportedCodecs codecs = ::media::EME_CODEC_AAC |
+                                  ::media::EME_CODEC_AVC1 |
+                                  ::media::EME_CODEC_VP9_PROFILE0 |
+                                  ::media::EME_CODEC_VP9_PROFILE2 |
+                                  ::media::EME_CODEC_VP8 |
+                                  ::media::EME_CODEC_OPUS |
+                                  ::media::EME_CODEC_VORBIS |
+                                  ::media::EME_CODEC_MPEG_H_AUDIO |
+                                  ::media::EME_CODEC_FLAC |
+                                  ::media::EME_CODEC_HEVC_PROFILE_MAIN  |
+                                  ::media::EME_CODEC_HEVC_PROFILE_MAIN10 |
+                                  ::media::EME_CODEC_AV1 |
+                                  ::media::EME_CODEC_AC3 |
+                                  ::media::EME_CODEC_EAC3;
+  return codecs;
+}
+
+void AddStarboardCmaKeySystems(::media::KeySystemInfos* key_system_infos) {
+  media::SupportedCodecs codecs = GetStarboardEmeSupportedCodecs();
```

Comment:
What should this function do on linux? If the answer is nothing, let's just put the whole body in the macro below

Reply:
done.